lib/Github/Api/PullRequest.php

```diff
@@ -22,16 +22,11 @@ class PullRequest extends AbstractApi
      *
      * @link https://developer.github.com/v3/pulls/#custom-media-types
      * @param string|null $bodyType
-     * @param string|null $apiVersion
      *
      * @return self
      */
-    public function configure($bodyType = null, $apiVersion = null)
```

Comment:
In that case I'll need to fix it since I realized when I was removing it that I never actually used the $apiVersion variable when setting the accept header value. :(